pdf_form_fill/pdf_form_fill.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    Copyright (C) 2010-2013 OpenERP s.a. (<http://openerp.com>).
+#    Copyright (C) 2013 initOS GmbH & Co. KG (<http://www.initos.com>).
+#    Author Nikolina Todorova <nikolina.todorova@initos.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.osv import orm
+from fdfgen import forge_fdf
```

Comment:
Someday someone is going to add a static folder with an icon, and then some people's installation will stop working because they don't have this library while this module is in their addons path. I think the best practice is to protect this import so that it does not fail upon module loading:

```
    try:
        from fdfgen import forge_fdf
    except ImportError:
        forge_fdf = False
```
Then, before using forge_fdf, you can check if forge_fdf is defined and raise otherwise at that moment.